search/signals.py

```diff
@@ -12,26 +12,32 @@
     Employment,
     Profile,
 )
+from dashboard.models import (
+    Certificate,
+    Enrollment,
+)
 from search.tasks import index_users, remove_user
 
 log = logging.getLogger(__name__)
 
 
 @receiver(post_save)
```

Comment:
oh... I see what's going on here: you filter afterwards. Still I think we should make a generic handler function and use it in specific signal handlers (one per sender).